test/fixtures.ts

```diff
@@ -168,3 +168,97 @@ export const buildRouteObject = ({
     state: 'NOT_INSURABLE',
   },
 })
+
+export const mockChainsResponse = [
```

Comment:
This should act as `msw` response.

Reply:
alright, making this change!

Reply:
@Abhikumar98 please don't resolve conversations until fixed 😄

Reply:
Now the mockResponse is being used in the msw mocks. Does this need to be relocated to another file?